omega/omega-transport/src/test/java/io/servicecomb/saga/omega/transport/UniqueIdGeneratorTest.java

```diff
@@ -0,0 +1,24 @@
+package io.servicecomb.saga.omega.transport;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Test;
+
+public class UniqueIdGeneratorTest {
+
+  private UniqueIdGenerator idGenerator = new UniqueIdGenerator();
+
+  @Test
+  public void nextIdIsUnique() {
+    Set<String> ids = new HashSet<>();
+    for (int i = 0; i < 10; i++) {
```

Comment:
does this test guarantee the id generator will generate unique ids in multithreaded scenario?